galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -78,6 +78,25 @@ var testGrid = []testCase{
 			{msg.MisplacedAnnotation, "Namespace staging"},
 		},
 	},
+	{
+		name:       "jwtTargetsInvalidServicePortName",
+		inputFiles: []string{"testdata/jwt-invalid-service-port-name.yaml"},
+		analyzer:   &auth.JwtAnalyzer{},
+		expected: []message{
+			{msg.JwtFailureDueToInvalidServicePortPrefix, "Policy default.my-namespace"},
+			{msg.JwtFailureDueToInvalidServicePortPrefix, "Policy default.my-namespace-2"},
```

Comment:
It would be good if these had names that made it obvious why they are failures/misconfigurations.

Reply:
I added some more unit tests and made sure that the names were more descriptive.